### ISSUE TITLE
API-level access to Yahoo Finance quotes data has been disabled

### ISSUE THREAD

**Issue by arrikhan** [iprak/yahoofinance#146]
Have noticed yahoo finance stopped working recently. URL in error logs takes me to a API feedback google form.

"We’re sorry for the inconvenience, but API-level access to Yahoo Finance quotes data has been disabled. "

`Received status 401 (Unauthorized User is unable to access this feature - https://bit.ly/yahoo-finance-api-feedback) for https://query1.finance.yahoo.com/v7/finance/quote?symbols=xxx.xx`

Has this killed this plugin?

**Comment by JmotoB** [comment]
Just noticed the same behavior and error in my instance. Restart of HA also seems to have resolved it for the time being.

**Comment by iprak** [comment]
Can you please include your locations? I am not seeing this from US.

**Comment by joelishness** [comment]
I don't know the reason (whether related to this API issue or not), but I am in the US and it stopped acquiring data 11/19 10:21:22 AM PDT.
Then this evening resolved by restarting Home Assistant

**Comment by iprak** [comment]
If this is still happening, then please include debug logging.

**Comment by torarnehelgesen** [comment]
Also have this problem. (europe - norway)

**Comment by torarnehelgesen** [comment]
This seems to be working again. fyi

**Comment by ffabinhoo** [comment]
Is it still working? 1.17.0 doesnt return anything

**Comment by iprak** [comment]
Yes it is working. I don't think there is any such release or did you mean 1.1.7 from 2022 ?
